tests/st/utils/data.py

```diff
@@ -260,7 +258,6 @@
         ],
         'ingress': [
             {
-                'notICMP': {'type': 19, 'code': 255},
```

Comment:
Why are we removing these? Which part of validation does this fail?

Reply:
There's a proto: TCP just below.

Reply:
These rules have both ICMP rules and Ports rules, which is invalid now:
- If includes port rules, protocol needs to be TCP. 
- If it includes ICMP rules, protocol needs to be ICMP